java/src/main/java/com/plaid/quickstart/resources/IndexResource.java

```diff
@@ -0,0 +1,45 @@
+package com.plaid.quickstart.resources;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.dropwizard.views.View;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+@Path("/")
+public class IndexResource {
+    private final String plaidEnvironment;
+    private final String plaidPublicKey;
+
+    public IndexResource(String _plaidEnvironment, String _plaidPublicKey) {
```

Comment:
we don't need underscores for parameters.

just do this.plaidEnvironment = plaidEnvironment

Reply:
done